.github/workflows/generate-docs-artifacts.yml

```diff
@@ -0,0 +1,67 @@
+# This GitHub action can generate documentation artifacts for a release when a tag is created.
+# Currently its setup to run on any tag that matches the pattern "v*" (ie. v0.1.0).
+
+# Documentation artifacts generated by this can action also be generated manually, but it is
+# encouraged to use the action on release events so that documentation stays up to date.
+# Manual process: Requires Nodejs and NPM
+# In the plugin root execute the command packer-docs-artifact command
+# > npx -p @hashicorp/packer-docs-artifacts generate
+#
+# The generated files will be placed under PLUGIN_ROOT/.doc-artifacts; this directory contains all the docs
+# and respective navigation information needed for including the plugin docs under packer.io/docs/
+#
+# Merge the generated files to the default branch for the plugin repository: main or master
+#
+# Once merged a one time pull-request will needs to be opened against hashicorp/packer to register the plugin docs.
+# This can be done by adding the block below for the respective plugins to the file website/data/docs-remote-navigation.js
```

Comment:
```suggestion
# This can be done by adding the block below for the respective plugins to the file website/data/docs-remote-navigation.js in the https://github.com/hashicorp/packer repository
```